packages/grafana-ui/src/components/Divider/Divider.mdx

```diff
@@ -5,22 +5,22 @@ import { Divider } from './Divider';
 
 # Divider
 
-Used to create separation between large sections of page content. Or used to separate smaller sections
-of components. i.e in the info section of the header.
+- Used to create separation between large sections of page content.
+- Used to separate smaller sections of components. i.e in the info section of the header.
```

Comment:
I would actually maybe remove this bit since it is repeated in the "when to use section"

Reply:
Ok, I made the suggested updates.thanks